added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/follow/CreateFollow.md

title: Following And Unfollowing
displayed_sidebar: developSidebar
---

<head>

>   <title>Create Following on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>Following And Unfollowing With The Subsocial JS SDK</title>
```
